Tests/js/models/assets/ez-usermodel-tests.js

```diff
@@ -28,9 +29,57 @@ YUI.add('ez-usermodel-tests', function (Y) {
                     "login": "admin",
                     "email": "dp@ez.no",
                     "enabled": "true",
-                    "name": "Administrator User"
+                    "name": "Administrator User",
+                    "Version":  {
+                        "Fields": {
+                            "field": [
+                                {
+                                    fieldDefinitionIdentifier: "first_name",
+                                    fieldValue: "Administrator",
+                                    id: 28,
+                                    languageCode: "eng-GB"
+                                },
+                                {
+                                    fieldDefinitionIdentifier: "image",
+                                    fieldValue: {
+                                        alternativeText: "",
+                                        fileName: "useravatar.png",
+                                        fileSize: 6216,
+                                        height: "48",
+                                        id: "0/8/1/0/180-6-eng-GB/useravatar.png",
+                                        imageId: "14-180",
+                                        inputUri: null,
+                                        path: "/0/8/1/0/180-6-eng-GB/useravatar.png",
+                                        uri: "/var/site/storage/images/0/8/1/0/180-6-eng-GB/useravatar.png",
+                                        width: "48"
+                                    },
+                                    id: 180,
+                                    languageCode: "eng-GB"
+                                },
+                            ]
+                        }
+                    }
+                },
+                "_contentType": this.contentType,
+                "document": {
+                    "User": {
+                        "ContentType": {
+                            "_href": 'some-rest-id'
+                        }
+                    }
                 }
             };
+            this.imageIdentifiers = ['image'];
+
+            Y.eZ.ContentType = Y.Model;
+
+            Y.Mock.expect(this.contentType, {
+                method: 'getFieldDefinitionIdentifiers',
+                args: [
+                    'ezimage'
+                ],
+                returns: this.imageIdentifiers
```

Comment:
so now the case where there's no image field in the User Content Type is not covered. Basically, you need to add a test where `getFieldDefinitionIdentifiers` returns and empty array and you check that `avatar` is null.